src/main/java/com/nike/cerberus/domain/SecureDataResponse.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.domain;
+
+public class SecureDataResponse {
+    private String requestId;
+    private String leaseId = "";
+    private boolean renewable = false;
+    private int leaseDuration = 3600;
+    private Object data;
+    private Object wrapInfo = null;
+    private Object warnings = null;
+    private Object auth = null;
+
+    public String getRequestId() {
+        return requestId;
+    }
+
+    public void setRequestId(String requestId) {
+        this.requestId = requestId;
+    }
+
+    public String getLeaseId() {
+        return leaseId;
+    }
+
+    public void setLeaseId(String leaseId) {
+        this.leaseId = leaseId;
+    }
+
+    public boolean isRenewable() {
+        return renewable;
+    }
+
+    public void setRenewable(boolean renewable) {
+        this.renewable = renewable;
+    }
+
+    public int getLeaseDuration() {
```

Comment:
Javadoc should say what unit of time everywhere, there were bugs in the past in the clients because people got confused.